test/unit/kubernetes-deploy/kubectl_test.rb

```diff
@@ -168,6 +168,14 @@ def test_run_with_raise_if_not_found_does_not_raise_on_other_errors
     build_kubectl.run("get", "pod", "foobar", raise_if_not_found: true)
   end
 
+  def test_run_output_is_sensitive_squashes_debug_logs
+    stub_open3(%W(kubectl get pods --namespace=testn --context=testc --request-timeout=#{timeout}),
+      resp: "", err: "oops", success: false)
+    logger.level = 0
```

Comment:
Interesting! I think we have a few tests that are manipulating the debug env var and rebuilding the logger (which prevents them from being parallelized) where they could be doing this instead. 🤔